pandas/core/groupby/groupby.py

```diff
@@ -1497,7 +1497,7 @@ def _bool_agg(self, val_test, skipna):
         Shared func to call any / all Cython GroupBy implementations.
         """
 
-        def objs_to_bool(vals: ArrayLike) -> tuple[np.ndarray, type]:
+        def objs_to_bool(vals: ArrayLike) -> tuple[ArrayLike, type]:
```

Comment:
this returns vals.view(np.int8). so np.ndarray is correct?

Reply:
Yes.  Changed it back.